docs/troubleshooting.md

```diff
@@ -0,0 +1,78 @@
+# Troubleshooting
+
+The Collector offers multiple ways to measure the health of the Collector
+as well as investigate issues.
+
+- By default, Prometheus metrics are exposed on port `8888` and path
+`/metrics`. A grafana dashboard can be found
+[here](https://grafana.com/grafana/dashboards/11575).
+
+- Logs can be helpful in identifying issues. The verbosity level, which
+defaults to `INFO` can also be adjusted by passing the `--log-level` flag
+to the `otelcol` process. See `--help` for more details.
+
+- [Local exporters](https://github.com/open-telemetry/opentelemetry-collector/tree/master/exporter#local-exporters)
+can be configured to inspect the data being processed by the Collector.
+
+- The [health_check](https://github.com/open-telemetry/opentelemetry-collector/blob/master/extension/README.md#health_check)
+extension, which by default is on port `13133`, can be used to ensure
+the Collector is functioning properly.
+
+- The [zpages](https://github.com/open-telemetry/opentelemetry-collector/blob/master/extension/README.md#zpages)
+extension, which by default is on port `55679`, can be used to check RPC
+stats via `/debug/rpcz` as well as trace data via `/debug/tracez`.
```

Comment:
RPC stats via `/debug/rpcz` are available only for gRPC and I don't think they have much value for typical users, not sure if it is worth mentioning. Anyway, at least clarify that `rpcz` is available for "some" receivers/exporters.